proposals/0095-AOA-multiplexing.md

```diff
@@ -1,72 +1,193 @@
 # AOA multiplexing similar to Android BT/SPP multiplexing
 
 * Proposal: [SDL-0095](0095-AOA-multiplexing.md)
-* Author: [Jennifer Hodges](https://github.com/jhodges55)
+* Author: [Jennifer Hodges](https://github.com/jhodges55), [Shinichi Watanabe](https://github.com/shiniwat)
 * Status: **Returned for Revisions**
 * Impacted Platforms: [ Android ]
 
 
 ## Introduction
 
-Add "multiplexing" to AOA transport. ![Overview](../assets/proposals/0095-AOA-multiplexing/0095-AOA-multiplexing_overview.png)
-This change introduces a better user experience as follows:
-- Multiple navigation apps can be selected by user without any operation on HS. It's same as iPhone's UX.
-- Non-navigation apps can use not only BT/SPP but AOA as transport. 
-- In the future, projection will be applied to not only navigation but also other category app then, multi "projection apps" can be supported.
- NOTICE: It will be depend on HU capability, "Whether multi projection app can be run in parallel (multi view port for each projection app) or not".
+This proposal aims to add AOA multiplexing capability to SdlRouterService.  
 
 ## Motivation
 
-Currently, only one AOA app can connect to SDL core because of AOA limitation. However, a few navigation apps will be released. And, video projection functionality will be opened for not only navigation but also other category apps. And not projection but using wide band-width and stable transport apps (ex. using high quality audio streaming or pass thru), may be considered in the future. This proposal is for supporting and realizing these requirements.
+Currently SdlRouterService works for MultiplexTransport, which supports Bluetooth transport as the underlying transport. This proposal extends SdlRouterService to support both Bluetooth and AOA (Android Open Accessory) transports, so that multiple SDL applications can share the AOA transport without having to worry about USB connection.
+AOA protocol basically allows only single application to access the USB accessory -- if multiple applications are designed to use the same accessory, Android system asks which application to use the accessory and shows a permission dialog. This proposal allows multiple SDL applications to share the USB accessory. 
```

Comment:
Please update first sentence to "AOA protocol basically allows only a single application to access... if multiple applications are designed to use the same accessory, the Android system asks which application should use the accessory..."